client/blocks/taxonomy-manager/list-item.jsx

```diff
@@ -105,9 +105,13 @@ class TaxonomyManagerListItem extends Component {
 		this.setState( { showTooltip: false } );
 	};
 
+	name = () => {
```

Comment:
`name` isn't technically a reserved word, but it is a property of functions in most JS implementations. I doubt this would break anything since it's only defined on the instance, but something like `getName` or `getTermName` might avoid confusion.